utils/io/fileutils/files.go

```diff
@@ -133,6 +133,32 @@ func ListFilesRecursiveWalkIntoDirSymlink(path string, walkIntoDirSymlink bool)
 	return
 }
 
+// Return the list of files in the specified path that has the extension
+func ListFilesWithSpecificExtension(path, ext string) ([]string, error) {
```

Comment:
I suggest shortening the function name to *ListFilesWithExtension*.

Reply:
Done